wpilibj/src/main/java/edu/wpi/first/wpilibj/smartdashboard/SendableBuilderImpl.java

```diff
@@ -251,7 +251,10 @@ public void addCloseable(AutoCloseable closeable) {
   @Override
   public void setSmartDashboardType(String type) {
     if (m_typePub == null) {
-      m_typePub = m_table.getStringTopic(".type").publish();
+      m_typePub =
+          m_table
+              .getStringTopic(".type")
+              .publishEx(StringTopic.kTypeString, "{\"SmartDashboard\":\"" + type + "\"}");
```

Comment:
I'm not a fan of the manual json string writing; it feels very fragile.
Isn't there a `setProperty` or something like that?

Reply:
Not for initial publishes, that has to be done after the topic is published.

Reply:
I didn’t want to add a JSON library dependency to the Java APIs, so they’re all string based currently.  It’s something we may want to revisit for 2027+.